tests/tensor_search/test_get_settings.py

```diff
@@ -22,10 +22,10 @@ def test_no_index(self):
     def test_default_settings(self):
         """default fields should be returned if index is created with default settings
             sample structure of output: {'index_defaults': {'treat_urls_and_pointers_as_images': False,
-                                          'text_preprocessing': {'split_method': 'sentence', 'split_length': 2,
-                                                                 'split_overlap': 0},
+                                          'text_preprocessing': {'splitMethod': 'sentence', 'split_length': 2,
```

Comment:
doesn't `text_preprocessing` need to be camelCase?

Reply:
Yes, it will be fixed in another PR (created index accept `snake_case`)